src/client/datascience/interactive-window/interactiveWindow.ts

```diff
@@ -316,8 +310,8 @@ export class InteractiveWindow implements IInteractiveWindowLoadable {
     }
 
     public dispose() {
-        if (this.kernel) {
-            this.kernel.dispose().ignoreErrors();
+        if (this.notebookController) {
+            this.notebookController.dispose();
         }
```

Comment:
Won't this remove the controller from the list of controller.
Not sure we want to dispose this.
It's shared with all notebooks